pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: wtf_sliding_sheet
 description: A widget that can be dragged and scrolled in a single gesture and snapped to a list of extents.
-version: 1.1.2
+version: 1.1.3
```

Comment:
Please update min flutter and Dart version in pubspec, because we introduced breaking changes